deploy/namespace.yaml

```diff
@@ -2,3 +2,5 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: openshift-windows-machine-config-operator
+  labels:
+    openshift.io/cluster-monitoring: "true"
```

Comment:
same here, missing newline (GH won't show single trailing new lines in this view, but will add the :no_entry: symbol if it's missing)